packages/editor/src/components/provider/index.js

```diff
@@ -30,6 +32,19 @@ class EditorProvider extends Component {
 		}
 	}
 
+	componentDidMount() {
+		if ( ! this.props.settings.styles ) {
+			return;
+		}
+		postcss( [ wrap( { selector: '.editor-block-list__block' } ) ] )
+			.process( this.props.settings.styles )
+			.then( ( css ) => {
+				const node = document.createElement( 'style' );
+				node.innerHTML = css;
```

Comment:
I know this is a POC but I assume we'd need to sanitise this (I'm not sure what kind of sanitisation, if any, `postcss` does). Just stating it so it's known 😄

Reply:
Maybe not, If a user installs a theme or a plugin injecting anything here, it's a cautious decision :). Plugin and themes can already do worse than injecting malicious CSS.

Reply:
Oh, I see!

Is there any reason not to escape it though? They shouldn't be putting anything here that isn't CSS… I guess I just always default to tinfoil-hat mode 😄